### PR TITLE
feat(meetings): locus info participant deltas

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
The scope of the changes in this pull request are to add the ability to determine which participants have changed their state between locus events.

Fixes #[SPARK-161919](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-161919)

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 1]
```diff
@@ -433,6 +434,62 @@ export default class LocusInfo extends EventsScope {
     }
   }
 
+  /**
+   * Update the participantDelta property of this object based on a list of
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js:438]: You are updating `this.deltaParticipants` in this function.

**Comment by adamweeks** [comment]
Please add some unit tests for this new function in `packages/node_modules/@webex/plugin-meetings/test/unit/spec/locus-info/index.js`

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js (hunk 1 above)
Comment (InteractiveTimmy) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js:438]: Will address this.

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ export default class LocusInfo extends EventsScope {
     this.membership = locus.membership || null;
     this.identities = locus.identities || null;
     this.participants = locus.participants || null;
+    this.deltaParticipants = [];
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js:57]: Could you add a comment above this line stating what the purpose of this object is? Make sure to include that it mentions that the data is only good for the last locus event.

**Pull request merged**